lib/formatter.js

```diff
@@ -130,17 +130,26 @@ function formatOrderedList(elem, fn, options) {
   var nonWhiteSpaceChildren = (elem.children || []).filter(function(child) {
     return child.type !== 'text' || !whiteSpaceRegex.test(child.data);
   });
+  // Return different functions for different OL types
+  var typeFunctions = {
+    1: function(start, i) { return i + 1 + start},
+    a: function(start, i) { return String.fromCharCode(i + start + 97)},
+    A: function(start, i) { return String.fromCharCode(i + start + 65)}
+  };
+  // Determine type
+  var olType = elem.attribs.type || '1'
   // Make sure there are list items present
   if (nonWhiteSpaceChildren.length) {
     // Calculate initial start from ol attribute
     var start = Number(elem.attribs.start || '1') - 1
     // Calculate the maximum length to i.
     var maxLength = (nonWhiteSpaceChildren.length + start).toString().length;
     _.each(nonWhiteSpaceChildren, function(elem, i) {
-      var index = i + 1 + start;
+      // Use different function depending on type
+      var index = typeFunctions[olType](start, i);
       // Calculate the needed spacing for nice indentation.
       var spacing = maxLength - index.toString().length;
-      var prefix = ' ' + index + '. ' + _s.repeat(' ', spacing);
+      var prefix = (olType === '1') ? ' ' + index + '. ' + _s.repeat(' ', spacing) : index + '. ';
```

Comment:
According to https://html.spec.whatwg.org/multipage/semantics.html#the-ol-element it looks like yes, they do continue to AA.  Doesn't fit my use case, but I added some commented out failing tests if someone wants to take it on, perhaps future me.